attributes/default.rb

```diff
@@ -4,7 +4,7 @@
 #
 # Copyright 2014-2016, Bloomberg Finance L.P.
 #
-
+default['go']['version'] = '1.7'
```

Comment:
Do we really need to hard-code this version here? 
It's highly undesirable for community cookbooks to override external attributes.